tests/reporter/kubernetes.go

```diff
@@ -144,14 +145,14 @@ func (r *KubernetesReporter) ReportSpec(specReport types.SpecReport) {
 }
 
 func (r *KubernetesReporter) DumpTestNamespaces(duration time.Duration) {
-	r.dumpNamespaces(duration, testsuite.TestNamespaces)
+	r.dumpTestObjects(duration, testsuite.TestNamespaces)
 }
 
 func (r *KubernetesReporter) DumpAllNamespaces(duration time.Duration) {
```

Comment:
Shouldn't we change these functions name as well?

Reply:
Good point. Done!